cmd/defaults.yaml

```diff
@@ -433,6 +433,7 @@ DefaultInstance:
         Type:
       Pat:
         ExpirationDate:
+        Scopes:
```

Comment:
I'm not sure about this added option. I don't see anywhere else in the PR code. Probably I'm missing something..

Reply:
deleted scope everywhere as it should not be configurable